shared/js/media/media_frame.js

```diff
@@ -137,6 +139,26 @@ MediaFrame.prototype.displayImage = function displayImage(blob,
   // It would be nice if users of this module could override this
   // background color.
   this.image.style.backgroundColor = '#222';
+  this.image.setAttribute('role', 'img');
+
+  this._localize = function _localize() {
```

Comment:
I'd prefer to have this code directly in the localize() method. Maybe define this.orientation here in displayImage(), and then put all of this localization code in MediaFrame.prototype.localize(). If you get the date from this.imageblob.lastModifiedDate (or whatever the File property is) and have the orientation, then I think the rest of this code does not need to be here.